src/bin/vip-slowlogs.ts

```diff
@@ -193,8 +193,15 @@ void command( {
 	module: 'slowlogs',
 	usage: baseUsage,
 } )
-	.option( 'limit', 'The maximum number of log entries', 500 )
-	.option( 'format', 'Output the log entries in CSV or JSON format', 'table' )
+	.option(
+		'limit',
+		'Set the maximum number of log entries. Accepts an integer value between 1 and 500.',
+		500
+	)
+	.option(
+		'format',
+		'Render output in a particular format. Accepts “table” (default), “csv”, and “json”.'
```

Comment:
3rd argument needs to be restored here too
```suggestion
		'Render output in a particular format. Accepts “table” (default), “csv”, and “json”.', 'table'
```

Reply:
Addressed here: https://github.com/Automattic/vip-cli/pull/2089/files#diff-70658fd60b34ff3f56894fed4f47f7cf337ff3c0895130afc9b6da9a22554c40R31